packages/app/src/cli/utilities/developer-platform-client/shopify-developers-client.ts

```diff
@@ -148,16 +148,20 @@ export class ShopifyDevelopersClient implements DeveloperPlatformClient {
 
   async appFromId(appIdentifiers: MinimalAppIdentifiers): Promise<OrganizationApp | undefined> {
     const {app} = await this.fetchApp(appIdentifiers)
-    const {modules} = app.activeRelease.version
```

Comment:
I don't think we need these changes. In the new backend, there won't be any apps without an active release version.